README.md

```diff
@@ -111,7 +111,7 @@ The included `deploy.js` is a script to automate building, signing, and deployin
 - Set the env var KEYCHAIN_PASSWORD to the keychain password of the current user
 - Copy the `deploy-config.sample.json` to `deploy-config.json` and edit the parameters accordingly. You'll need a HockeyApp account to get ids and keys
 - Put any Android keystore files into `edge-react-gui/keystores/`
-- If using Firebase, put your account's `google-services.json` and `GoogleService-Info.plist` into `edge-react-gui/`
+- If using Firebase, put your account's `google-services.json` and `GoogleService-Info.plist` into `edge-react-gui/android/app` and `edge-react-gui/ios/edge`
```

Comment:
These instructions actually stay the same, since deploy.js includes the following lines:

```js
  if (fs.existsSync(`${buildObj.guiDir}/google-services.json`)) {
    call(`cp -a ${buildObj.guiDir}/google-services.json ${buildObj.guiPlatformDir}/app/`)
  }
```

So Jenkins just drops the files in the root directory, and the deploy.js script moves them to the platform-specific folders.

If we want to remove this logic from the deploy script, we *could* do something like this in the prepare script:

```sh
if [ -f GoogleService-Info.plist ]; then
  cp GoogleService-Info.plist ios/edge/GoogleService-Info.plist
else
   cp ios/edge/GoogleService-Info.sample.plist ios/edge/GoogleService-Info.plist
fi
```

That means the app with build with either the sample file or the user-supplied credentials from the root directory. You would have to add the root-directory copies to .gitignore as well.

Reply:
I force-pushed the commit removing the readme change. Let's keep everything else the same to avoid mucking around with too many things